src/app/bav/fields.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
I'm confused why do we need this if we're getting nameParts from IPV and CIC have already done the same validation on name entry?

Reply:
Yep good point, have removed the validations and but left the `fields.js` file with placeholder values as it is required for the form-wizard setup and assume we will need field validations for bank account info?